src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -0,0 +1,271 @@
+)
+@click.option(
+    "--debug",
+    default=False,
+    type=bool,
+    is_flag=False,
+    flag_value=True,
+    help="""Set PyMAPDL to prints the debug logging to the console output.""",
+)
+def submit(
+    main_file: str,
+    name: str = None,
+    url: str = None,
+    user: str = None,
+    password: str = None,
+    python: float = 3.9,
+    output_files: Optional[Union[str, list]] = None,
+    shell_file: str = None,
+    requirements_file: str = None,
+    extra_files: Optional[Union[str, list]] = None,
+    config_file: str = None,
+    num_cores: int = None,
+    memory: int = None,
+    disk_space: int = None,
+    exclusive: bool = None,
+    max_execution_time: int = None,
+    wait: bool = False,
+    save_config_file: bool = False,
+    debug: bool = False,
+):
+    """Example code:
+    pymapdl submit my_file.sh my_file_01.py my_file_02  --name="my job" --url="https://10.231.106.91:3000/hps" --user=repuser --password=repuser --python=3.9
+    """
+    import json
+
+    from ansys.mapdl.core.hpc.pyhps import (
+        create_pymapdl_pyhps_job,
+        get_value_from_json_or_default,
+        wait_for_completion,
+    )
+
+    if debug:
+        logging.basicConfig(
+            format="[%(asctime)s | %(levelname)s] %(message)s", level=logging.DEBUG
+        )
+
+    if config_file is None:
+        config_file = os.path.join(os.getcwd(), "hps_config.json")
+        logger.debug(f"Using default hps configuration file: {config_file}")
+
+    url = get_value_from_json_or_default(url, config_file, "url", None)
+    user = get_value_from_json_or_default(user, config_file, "user", None)
+    password = get_value_from_json_or_default(password, config_file, "password", None)
+    python = get_value_from_json_or_default(python, config_file, "python", 3)
+    name = get_value_from_json_or_default(name, config_file, "name", "My PyMAPDL job")
+
+    num_cores = get_value_from_json_or_default(num_cores, config_file, "num_cores", 1)
+    memory = get_value_from_json_or_default(memory, config_file, "memory", 100)
+    disk_space = get_value_from_json_or_default(
+        disk_space, config_file, "disk_space", 100
+    )
+    exclusive = get_value_from_json_or_default(
+        exclusive, config_file, "exclusive", False
+    )
+    max_execution_time = get_value_from_json_or_default(
+        max_execution_time, config_file, "max_execution_time", 0
+    )
+
+    proj, _ = create_pymapdl_pyhps_job(
+        main_file=main_file,
+        name=name,
+        url=url,
+        user=user,
+        password=password,
+        python=python,
+        output_files=output_files,
+        shell_file=shell_file,
+        requirements_file=requirements_file,
+        extra_files=extra_files,
+        config_file=config_file,
+        num_cores=num_cores,
+        memory=memory,
+        disk_space=disk_space,
+        exclusive=exclusive,
+        max_execution_time=max_execution_time,
+    )
+
+    if save_config_file:
```

Comment:
@germa89 ideally there could be multiple configurations that can be saved so that users can switch between clusters easily.

Reply:
You can easily change the file path in ``config_file`` to point to other configurations